src/Report/JunitReporter.php

```diff
@@ -0,0 +1,145 @@
+        /** @var \DomElement $testsuite */
+        $testsuite = $testsuites->appendChild($dom->createElement('testsuite'));
+        $testsuite->setAttribute('name', 'PHP CS Fixer report');
+
+        if (count($reportSummary->getChanged())) {
+            $this->createFailedTestCases($dom, $testsuite, $reportSummary);
+        } else {
+            $this->createSuccessTestCase($dom, $testsuite);
+        }
+
+        if ($reportSummary->getTime()) {
+            $testsuite->setAttribute(
+                'time',
+                sprintf(
+                    '%.3f',
+                    $reportSummary->getTime() / 1000
+                )
+            );
+        }
+
+        $dom->formatOutput = true;
+
+        return $dom->saveXML();
+    }
+
+    private function createSuccessTestCase(\DOMDocument $dom, \DOMElement $testsuite)
+    {
+        $testcase = $dom->createElement('testcase');
+        $testcase->setAttribute('name', 'All OK');
+        $testcase->setAttribute('assertions', 1);
+
+        $testsuite->appendChild($testcase);
+        $testsuite->setAttribute('tests', 1);
+        $testsuite->setAttribute('assertions', 1);
+        $testsuite->setAttribute('failures', 0);
+        $testsuite->setAttribute('errors', 0);
+    }
+
+    /**
+     * @param \DOMDocument  $dom
+     * @param \DOMElement   $testsuite
+     * @param ReportSummary $reportSummary
+     */
+    private function createFailedTestCases(\DOMDocument $dom, \DOMElement $testsuite, ReportSummary $reportSummary)
+    {
+        $assertionsCount = 0;
+        foreach ($reportSummary->getChanged() as $file => $fixResult) {
+            $testcase = $this->createFailedTestCase(
+                $dom,
+                $file,
+                $fixResult,
+                $reportSummary->shouldAddAppliedFixers()
+            );
+            $testsuite->appendChild($testcase);
+            $assertionsCount += (int) $testcase->getAttribute('assertions');
+        }
+
+        $testsuite->setAttribute('tests', count($reportSummary->getChanged()));
+        $testsuite->setAttribute('assertions', $assertionsCount);
+        $testsuite->setAttribute('failures', $assertionsCount);
+        $testsuite->setAttribute('errors', 0);
+    }
+
+    /**
+     * @param \DOMDocument $dom
+     * @param string       $file
+     * @param array        $fixResult
+     * @param bool         $addAppliedFixers
+     *
+     * @return \DOMElement
+     */
+    private function createFailedTestCase(\DOMDocument $dom, $file, array $fixResult, $addAppliedFixers)
+    {
+        $appliedFixersCount = count($fixResult['appliedFixers']);
+
+        $testName = str_replace('.', '_DOT_', preg_replace('@\.'.pathinfo($file, PATHINFO_EXTENSION).'$@', '', $file));
+
+        $testcase = $dom->createElement('testcase');
+        $testcase->setAttribute('name', $testName);
+        $testcase->setAttribute('class', $testName);
+        $testcase->setAttribute('file', $file);
```

Comment:
I found some schema versions
* https://svn.jenkins-ci.org/trunk/hudson/dtkit/dtkit-format/dtkit-junit-model/src/main/resources/com/thalesgroup/dtkit/junit/model/xsd/
* https://github.com/gabrielfalcao/lettuce/blob/master/tests/functional/xunit.xsd
* https://windyroad.com.au/dl/Open%20Source/JUnit.xsd ([bamboo uses it](https://confluence.atlassian.com/jirakb/junit-parser-task-in-bamboo-shows-failed-to-parse-test-result-file-error-when-parsing-test-results-file-800701931.html))

Now I think should use latest (http://help.catchsoftware.com/display/ET/JUnit+Format)